app/jobs/google_scraping_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'httparty'
+require 'nokogiri'
+
+class GoogleScrapingJob < ApplicationJob
+  # default retry count is 5
```

Comment:
Where is the 5 comes from @gutakk 🤔

Reply:
5 is default retry attempts of the `retry_on` method
Ref : https://edgeapi.rubyonrails.org/classes/ActiveJob/Exceptions/ClassMethods.html#method-i-retry_on

Reply:
Update comment to be more clearer in d40a68be796eae956b7649273128f8390f9c8385